apiserversdk/proxy.go

```diff
@@ -151,6 +151,17 @@ func (rrt *retryRoundTripper) RoundTrip(req *http.Request) (*http.Response, erro
 		}
 
 		if apiserversdkutil.IsSuccessfulStatusCode(resp.StatusCode) {
+			if resp.Body != nil {
```

Comment:
Nice @CheyuWu! But at first glance, resp has just been created, so it should be impossible for it to have already been drained and closed. How could that happen?